cypress/e2e/payments.spec.js

```diff
@@ -4,29 +4,7 @@ describe('The Model Plan Payment Form', () => {
   });
 
   it('completes a Model Plan Ops Eval and Learning form', () => {
-    cy.visit('/');
-
-    cy.contains('a', 'Start a draft model plan').click();
-
-    cy.contains('h1', 'Start a new model plan');
-
-    cy.get('[data-testid="continue-link"]').click();
-
-    // General Model Plan Information
-
-    cy.get('#new-plan-model-name')
-      .type('My New Model Plan')
-      .should('have.value', 'My New Model Plan');
-
-    cy.contains('button', 'Next').click();
-
-    cy.location().should(loc => {
-      expect(loc.pathname).to.match(/\/models\/new-plan\/.{36}\/collaborators/);
-    });
-
-    cy.get('[data-testid="continue-to-tasklist"]').click();
-
-    cy.contains('h1', 'Model Plan task list');
+    cy.clickPlanTableByName('Empty Plan');
```

Comment:
@garyjzhao I ripped out this existing code that started an empty model plan in favor of using the pre-seeded empty model plan (which I actually had to fix locally - perhaps you ran into this locally too!)